core/field_dropdown.js

```diff
@@ -179,7 +180,7 @@ Blockly.FieldDropdown.prototype.showEditor_ = function() {
       /** @type {!Element} */ (this.menu_.getElement()), 'blocklyDropdownMenu');
 
   this.positionMenu_(this.menu_);
-
+  Blockly.utils.style.scrollIntoContainerView(this.selectedMenuItem_.getElement(), this.menu_.getElement());
```

Comment:
```
if (this.selectedMenuItem_) {
...
```

Reply:
Travis is also failing on eslint. That's because this exceeds the line length, change to this: 
```
  // Scroll the dropdown to show the selected menu item.
  if (this.selectedMenuItem_) {
    Blockly.utils.style.scrollIntoContainerView(
        this.selectedMenuItem_.getElement(), this.menu_.getElement());
  }
```

Reply:
Done!